socorro/lib/search_common.py

```diff
@@ -292,24 +294,37 @@ def fix_date_parameter(self, parameters):
     def fix_process_type_parameter(parameters):
         """Correct the process_type parameter.
 
-        If process_type is 'browser', replace it with a 'does not exist'
-        parameter. Do nothing in all other cases.
+        Previously, crash reports that didn't have a ProcessType annotation
```

Comment:
We can remove this staticmethod when we remove all the "browser" shenanigans, but for now it helps us bridge the gap between the old way and the new way.